packages/dev-server/README.md

```diff
@@ -0,0 +1,70 @@
+# Dev server
+
+> Part of Open Web Component Recommendation [open-wc](https://github.com/open-wc/open-wc/)
+
+Open Web Components provides a set of defaults, recommendations and tools to help facilitate your Web Component. Our recommendations include: developing, linting, testing, tooling, demoing, publishing and automating.
```

Comment:
```suggestion
Open Web Components provides a set of defaults, recommendations and tools to help facilitate your web component project. Our recommendations include: developing, linting, testing, building, tooling, demoing, publishing and automating.
```

Reply:
I copied it from testing-karma. Let's do a sweep of all the packages :P

Reply:
i did it in https://github.com/open-wc/open-wc/pull/154